components/camel-jsonata/src/generated/resources/org/apache/camel/component/jsonata/jsonata.json

```diff
@@ -0,0 +1,37 @@
+{
+  "component": {
+    "kind": "component",
+    "name": "jsonata",
+    "title": "JSONATA",
+    "description": "JSON to JSON transformation using JSONATA.",
+    "deprecated": false,
+    "firstVersion": "3.4.0",
```

Comment:
You need to regenerate the src/generated.. it's 3.5.0

Reply:
Done.